is.js

```diff
@@ -296,6 +296,55 @@
     // is a given number infinite?
     is.infinite = not(is.finite);
 
+    // is a given number zero?
+    is.zero = function(numb) {
+        return is.number(numb) && numb === 0;
+    };
+
+    // is a given number one?
+    is.one = function(numb) {
+        return is.number(numb) && numb === 1;
+    };
+
+    // is a given number negative or positive but no zero?
+    is.nonZero = not(is.zero);
+
+    // is a given number non-positive?
+    is.nonPositive = function(numb) {
+        return is.negative(numb) || numb === 0;
+    }
+
+    // is a given number non-negative?
+    is.nonNegative = function(numb) {
+        return is.positive(numb) || numb === 0;
+    }
+
+    // is a given number perfect square?
+    // Source https://en.wikipedia.org/wiki/Triangular_number
```

Comment:
Yep, it's fixed in [latest commit](https://github.com/arasatasaygin/is.js/pull/194/commits/5d158be48659441e93ba0501d47fc8d46b3240b8)